app/admin/arbre_helpers.rb

```diff
@@ -274,4 +274,37 @@ def self.attachments_list(context, attachments)
       end
     end
   end
+
+  def self.affinity_card(context, affinity)
+    context.instance_eval do
+      from = if affinity.person == self.resource
```

Comment:
En esta estructura, from y to pueden ser métodos de affinity, que reciban al otro user y devuelvan al que corresponde.
También, no se si hace falta distinguir entre 'from y to',
podría autoritativamente primero mostrar al resource actual, y después tener un método que busque a la contraparte, y el nombre de la relación, en affinity, en base al usuario que le pasamos.
ej:
from = self.resource
to = affinity.other(self.resource)
relationship_label = affinity.label(self.resource) # spouse / spouse_of